packages/frontend/src/components/RefreshServer/RefreshStepsModal/RefreshStepsModal.style.tsx

```diff
@@ -0,0 +1,91 @@
+import { Colors, H4, Icon } from '@blueprintjs/core';
```

Comment:
File name missing an **s** in **styles**. Same for the other file in this PR.

RefreshStepsModal.styles.tsx